snake.js

```diff
@@ -63,20 +63,18 @@ function draw(){
         y : newCoords[1]
     }
 
-    snake.unshift(tale);
+    if(snakeCollision(tale, snake) === true){
```

Comment:
It's redundant to make explicit comparison === true. Just `if (snakeCollision(tale, snake)) {` and it will work as expected.

Reply:
And remember about spaces. Your code looks ugly.